exchange/utils.go

```diff
@@ -805,3 +805,11 @@ func mergeBidderRequests(allBidderRequests []BidderRequest, bidderNameToBidderRe
 	}
 	return allBidderRequests
 }
+
+func BuildExtData(data []byte) []byte {
```

Comment:
This function name seems a little vague. Perhaps WrapJSONInData() as this is not specific to .ext fields, it just places the argument in a JSON `data` object.